toolsWebsite/tools/templates/tools/index.html

```diff
@@ -51,7 +51,7 @@
                   </a>
                   <div class="dropdown-menu dropdown-menu-right" aria-labelledby="cartDropdown">
                     <div id="cartDropdownItems"></div>
-                    <a href="/checkout">
+                    <a href="./checkout">
```

Comment:
why the `./checkout` instead of `/checkout`?

Reply:
Having /checkout leaves off the tools and tools/checkout was giving me tools/tools/checkout.  ./ gave me tools/checkout.

Reply:
what about just `checkout` or `/tools/checkout`? haha I think that was the way I was doing it.

Reply:
I changed it to /tools/checkout so its consistent